Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -519,7 +519,8 @@ BTDiagnostics::DefineCellCenteredMultiFab(int lev)
 #else
     const int ncomps = static_cast<int>(m_cellcenter_varnames.size());
 #endif
-    WarpX::AllocInitMultiFab(m_cell_centered_data[lev], ba, dmap, ncomps, amrex::IntVect(ngrow), lev, "cellcentered_BTD", 0._rt);
+    m_cell_centered_data[lev] = std::make_unique<amrex::MultiFab>(ba, dmap, ncomps, amrex::IntVect(ngrow));
```

Comment:
Why not use  `MultiFabRegister::alloc_init`?

Reply:
Technically, we deallocated this when the `BTDiagnostics` is deallocated. This currently probably happens only at the end of the simulation, so we do not need to call `::erase` even. If there is an earlier deallocation or more logic in `~BTDiagnostics` then we should call `::erase` in it.

For the name, we should make a unique name for the current BTD diagnostics name (from the inputs) and the station it works on. This will be nice, because we can now more easily track the overhead of BTD diags.

Reply:
The lifetime of the diagnostics is the same as the lifetime of the WarpX class so the erase is not needed.